TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/ResultsToLogsConsumer.java

```diff
@@ -107,8 +107,10 @@ private Map<String, Object> toJSONObject(SourceTargetCaptureTuple tuple, ParsedH
      */
     public void accept(SourceTargetCaptureTuple tuple, ParsedHttpMessagesAsDicts parsedMessages) {
         final var index = tupleCounter.getAndIncrement();
-        progressLogger.atInfo().setMessage(()->toTransactionSummaryString(index, tuple, parsedMessages)).log();
-        tupleLogger.atInfo().setMessage(() -> {
+        progressLogger.atInfo().setMessage("{}")
+            .addArgument(()->toTransactionSummaryString(index, tuple, parsedMessages))
+            .log();
+        tupleLogger.atInfo().setMessage("{}").addArgument(() -> {
             try {
```

Comment:
While unrelated to this change set, this log message looks suspect - if there is an exception in the parsing system it will only show up if the log level is set to info.

Reply:
Fixed this in cases an exception is thrown in parsing